### PR TITLE
EL9 initial templates

### PR DESCRIPTION
**Pull request by jouvin**

### CODE REVIEW

**Comment by jouvin** [comment]
@jrha I realized I did a big mess with an unfortunate push to EL9 branch. And I misconfigured the target branch in this PR... I'll sort out everything later today (tonight) as I am unavailable for the rest of the afternoon...

**Comment by jrha** [comment]
I should be able to reset the branches.

File: rpms/base.pan [hunk 1]
```diff
@@ -28,8 +25,8 @@ variable OS_CORE_RDMA_ENABLED ?= false;
     pkg_repl('mcelog');
     pkg_repl('microcode_ctl');
     pkg_repl('NetworkManager');
-    # network-scripts is required by ncm-network 21.4.0
-    pkg_repl('network-scripts');
+    # ncm-network relies on ifcfg scripts
+    pkg_repl('NetworkManager-initscripts-updown');
```
Comment (jrha) [inline rpms/base.pan:29]: While this is true, we're not supporting the `initscripts` back-end on EL9, so this isn't the correct justification.

File: rpms/base.pan [hunk 2]
```diff
@@ -58,6 +65,10 @@ variable OS_CORE_RDMA_ENABLED ?= false;
   # ncm-spma will keep trying removing all kernels and only the running one will be kept.
   pkg_repl('kernel');
 
+  # chkconfig: required by several Quattor components
```
Comment (jrha) [inline rpms/base.pan:68]: Hmm, we should have a list of which require it.

File: config/core/base.pan [hunk 3]
```diff
@@ -16,16 +16,16 @@ variable SITE_ADDITIONAL_PACKAGES ?= undef;
...
-    "arch",         "x86_64"
+    "version",      "el9x",
+    "arch",         "x86_64",
 );
 
-variable RPM_BASE_FLAVOUR = '8';
-variable RPM_BASE_FLAVOUR_VERSIONID = 8;
+variable RPM_BASE_FLAVOUR = '9';
+variable RPM_BASE_FLAVOUR_VERSIONID = 9;
 variable RPM_BASE_FLAVOUR_NAME = format('el%s',RPM_BASE_FLAVOUR_VERSIONID);
```
Comment (jrha) [inline config/core/base.pan:29]: Should these all be `final`?

Comment (jouvin) [reply config/core/base.pan:29]: I think almost all variables in this template should be `final`. There is an historical lack of `final` statements in the template library. Let's start by fixing this one!

File: rpms/base.pan (hunk 2 above)
Comment (jouvin) [reply rpms/base.pan:68]: I think it is misworded. Components should be interpreted in the generic meaning, part of Quattor. I don't think any configuration module still requires it but we should check in grid components may be. As for the `chkconfig` usage in various templates, I think I cleaned most of the template library now (but not sure if I upstreamed everything). At least EL9 OS templates should be free of it. For the short term I suggest to rephrase the comment.

**Comment by jouvin** [comment]
Looks good!

**Review by jrha — DISMISSED** [review]

**Comment by jouvin** [comment]
I just pushed modifications removing `initscripts` by default (with a variable to help the transition if somebody really needs it but probably something we can remove later) and added a variable to disable installation of `chkconfig` and `ncm-chkconfig` that must be kept to `true` until `ncm-named` and `ncm-systemd` no longer require it.

**Review by jrha — APPROVED** [review]

**Pull request merged**